haystack/nodes/extractor/entity.py

```diff
@@ -185,3 +489,347 @@ def simplify_ner_for_qa(output):
 
         compact_output.append({"answer": answer.answer, "entities": entities})
     return compact_output
+
+
+class _EntityPostProcessor:
+    """This class is used to conveniently collect all functions related to the postprocessing of entity extraction.
```

Comment:
Hey @vblagoje and @masci I ended up using HuggingFace's postprocessing source code from their `TokenClassificationPipeline` and then made a fair amount of modifications for it to work in all of our use cases. To comply with [HuggingFace's license](https://github.com/huggingface/transformers/blob/main/LICENSE) I wanted to ask how should I reproduce their license and provide notice that we are using a derivative of their work?

Reply:
Following the format from `haystack/modeling/model/language_model.py` https://github.com/deepset-ai/haystack/blob/797c20c966fe46308f646e02d662ca87155a9d4a/haystack/modeling/model/language_model.py#L1-L19
I've added the following lines to the top of `entity.py` https://github.com/deepset-ai/haystack/blob/d8ad3e1fe5d8560da2a4f82cae8a8bf61c8ac42b/haystack/nodes/extractor/entity.py#L1-L18

Reply:
@sjrl that's the correct way to comply to the original license 👍

Reply:
Perfect! Thanks for the confirmation.